### PR TITLE
logging: adds logentries endpoint

### PR DESCRIPTION
**Pull request by mccurdyc**
## Relevant (Referenced) Link(s)

* [Fastly API Docs](https://docs.fastly.com/api/logging#api-section-logging_logentries)
* [fastly/go-fastly](https://github.com/fastly/go-fastly/blob/88b210b97743b0bfece81dc3dbbd7e3254e22d97/fastly/logentries.go)

## Validation

```
$ make test
```

```
git clone git@github.com:mccurdyc/cli.git /tmp/cli && \
( 
    cd /tmp/cli && \
    git checkout mccurdyc/logging-logentries && \
    make test && \
    rm -rf /tmp/cli \
)
```

### CODE REVIEW

File: pkg/logging/logentries/create.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("create", "Create an Logentries logging endpoint on a Fastly service version").Alias("add")
+
+	c.CmdClause.Flag("name", "The name of the Logentries logging object. Used as a primary key for API access").Short('n').Required().StringVar(&c.Input.Name)
+	c.CmdClause.Flag("service-id", "Service ID").Short('s').StringVar(&c.manifest.Flag.ServiceID)
+	c.CmdClause.Flag("version", "Number of service version").Required().IntVar(&c.Input.Version)
+
+	c.CmdClause.Flag("port", "The port number").Required().UintVar(&c.Input.Port)
+	c.CmdClause.Flag("use-tls", "Whether to use TLS for secure logging. Can be either true or false").BoolVar(&c.useTLS)
```
Comment (mccurdyc) [inline pkg/logging/logentries/create.go:37]: Related discussion - https://github.com/fastly/cli/pull/55#discussion_r421071174

File: pkg/logging/logentries/describe.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/go-fastly/fastly"
+)
+
+// DescribeCommand calls the Fastly API to describe an Logentries logging endpoint.
```
Comment (phamann) [inline pkg/logging/logentries/describe.go:14]:
```suggestion
// DescribeCommand calls the Fastly API to describe a Logentries logging endpoint.
```

File: pkg/logging/logentries/create.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	// so they can be casted to go-fastly's custom `Compatibool` type later.
+	useTLS bool
+}
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent common.Registerer, globals *config.Data) *CreateCommand {
+	var c CreateCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("create", "Create an Logentries logging endpoint on a Fastly service version").Alias("add")
```
Comment (phamann) [inline pkg/logging/logentries/create.go:30]:
```suggestion
	c.CmdClause = parent.Command("create", "Create a Logentries logging endpoint on a Fastly service version").Alias("add")
```

File: pkg/logging/logentries/root.go [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+type RootCommand struct {
+	common.Base
+	// no flags
+}
+
+// NewRootCommand returns a new command registered in the parent.
+func NewRootCommand(parent common.Registerer, globals *config.Data) *RootCommand {
+	var c RootCommand
+	c.Globals = globals
+	c.CmdClause = parent.Command("logentries", "Manipulate Fastly service version Logentries logging endpoints.")
```
Comment (phamann) [inline pkg/logging/logentries/root.go:21]:
```suggestion
	c.CmdClause = parent.Command("logentries", "Manipulate Fastly service version Logentries logging endpoints")
```

File: pkg/logging/logentries/create.go [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+	var c CreateCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("create", "Create a Logentries logging endpoint on a Fastly service version").Alias("add")
+
+	c.CmdClause.Flag("name", "The name of the Logentries logging object. Used as a primary key for API access").Short('n').Required().StringVar(&c.Input.Name)
+	c.CmdClause.Flag("service-id", "Service ID").Short('s').StringVar(&c.manifest.Flag.ServiceID)
+	c.CmdClause.Flag("version", "Number of service version").Required().IntVar(&c.Input.Version)
+
+	c.CmdClause.Flag("port", "The port number").Required().UintVar(&c.Input.Port)
```
Comment (mccurdyc) [inline pkg/logging/logentries/create.go:36]: @mccurdyc TODO: I don't think that this should be required by the CLI API.

**Pull request merged**